modules/dcache/src/main/java/diskCacheV111/services/PoolStatisticsV0.java

```diff
@@ -241,13 +183,14 @@ protected SimpleDateFormat initialValue()
 
     private final CellNucleus      _nucleus;
 
-    private CellCron.TimerTask _hourly;
+    private final ScheduledExecutorService executor = Executors.newScheduledThreadPool(1);
+    private final Runnable hourlyRunner = new HourlyRunner();
+    private Calendar _calendar;
```

Comment:
Better name the Calendar object "lastRun" or "previousRun" to make clear what it is used for. The current name does not convey anything.